src/main/resources/data.sql

```diff
@@ -1,35 +1,63 @@
-VALUES ('Classic Toy Car', NULL, (SELECT id FROM member WHERE nickname = 'userTwo'), '2024-07-01', 'VEHICLES', 'A classic toy car from the 80s.', 'LIKE_NEW', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Educational Board Game Set', NULL, (SELECT id FROM member WHERE nickname = 'userOne'), '2024-07-05', 'EDUCATIONAL_TOYS', 'A complete educational board game set with all pieces.', 'GOOD', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Electronic Stuffed Animal Collection', NULL, (SELECT id FROM member WHERE nickname = 'userTwo'), '2024-07-10', 'ELECTRONIC_TOYS', 'A collection of electronic stuffed animals in various sizes.', 'USED', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Lego Building Set', NULL, (SELECT id FROM member WHERE nickname = 'userOne'), '2024-07-15', 'CONSTRUCTION_TOYS', 'A large Lego building set with instructions.', 'LIKE_NEW', 'AVAILABLE');
-
--- Additional listings
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Antique Doll', NULL, (SELECT id FROM member WHERE nickname = 'userTwo'), '2024-07-20', 'DOLLS', 'An antique doll in excellent condition.', 'LIKE_NEW', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Remote Control Car', NULL, (SELECT id FROM member WHERE nickname = 'userOne'), '2024-07-25', 'VEHICLES', 'A remote control car with all accessories.', 'GOOD', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Educational Puzzle Set', NULL, (SELECT id FROM member WHERE nickname = 'userTwo'), '2024-07-30', 'EDUCATIONAL_TOYS', 'A set of educational puzzles for kids.', 'USED', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Vintage Train Set', NULL, (SELECT id FROM member WHERE nickname = 'userOne'), '2024-08-05', 'OTHER', 'A vintage train set with tracks and accessories.', 'GOOD', 'AVAILABLE');
-
-INSERT INTO listing (title, photo, member_id, date_posted, category, description, condition, status_listing)
-VALUES ('Electronic Learning Tablet', NULL, (SELECT id FROM member WHERE nickname = 'userTwo'), '2024-08-10', 'ELECTRONIC_TOYS', 'An electronic learning tablet for kids.', 'BRAND_NEW', 'AVAILABLE');
\ No newline at end of file
+INSERT INTO listing (title, member_id, date_posted, category, description, condition, status_listing)
+VALUES ('Vintage Action Figure', (SELECT id FROM member WHERE nickname = 'userOne'), '2024-06-27', 'ACTION_FIGURES', 'A vintage action figure in good condition.', 'GOOD', 'AVAILABLE');
```

Comment:
I may edit this script soon to add more detailed and meaningful data if that's ok? This will help the front-end look more realistic/full.